### PR TITLE
Owners popular integrations

### PR DESCRIPTION
**Pull request by bakatzir**
fixes: https://github.com/demisto/etc/issues/33328

I followed https://datastudio.google.com/u/0/reporting/b29f9806-7f0b-4b47-b7a0-0f139a40eb47/page/LafpB?s=nSXy4e1KKR4 to decide for top6.

I also thought of adding panw integrations, please ±1 per line your thoughts, as we do not want to abuse the codeowners file.
PAN-OS
AutoFocusv2
WildFirev2
CortexXDR
I myself did not add them even though they were top20, as we(yet) to break BC for them IIRC.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,9 @@
 /Tests/Marketplace/approved_tags.json @demisto/content-leaders @michalgold
 /Tests/Marketplace/approved_usecases.json @demisto/content-leaders @michalgold
 /Packs/CommonScripts/Scripts/SetGridField/* @yaakovi
+/Packs/Jira/Integrations/JiraV2/* @demisto/content-leaders
+/Packs/ServiceNow/Integrations/ServiceNowv2/* @demisto/content-leaders
+/Packs/EWS/Integrations/EWSv2/* @demisto/content-leaders
+/Packs/Slack/Integrations/Slack/* @demisto/content-leaders
+/Packs/SplunkPy/Integrations/SplunkPy/* @demisto/content-leaders
+/Packs/Active_Directory_Query/Integrations/Active_Directory_Query/* @demisto/content-leaders
```
Comment (ShahafBenYakir) [inline .github/CODEOWNERS:9]:
```suggestion
# Security related
/Tests/Marketplace/approved_tags.json @demisto/content-leaders @michalgold
/Tests/Marketplace/approved_usecases.json @demisto/content-leaders @michalgold

# Importamt integrations
/Packs/Jira/Integrations/JiraV2/* @demisto/content-leaders
/Packs/ServiceNow/Integrations/ServiceNowv2/* @demisto/content-leaders
/Packs/EWS/Integrations/EWSv2/* @demisto/content-leaders
/Packs/Slack/Integrations/Slack/* @demisto/content-leaders
/Packs/SplunkPy/Integrations/SplunkPy/* @demisto/content-leaders
/Packs/Active_Directory_Query/Integrations/Active_Directory_Query/* @demisto/content-leaders

# Importamt scripts
/Packs/CommonScripts/Scripts/SetGridField/* @yaakovi
```

**Comment by yaakovi** [comment]
it's a start but not fully resolves the issue

**Review by ShahafBenYakir — APPROVED** [review]

**Comment by ShahafBenYakir** [comment]
@bakatzir Per @yaakovi's comment I think it is better to remove the "fixes" from the PR body, so we will still have the issue.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
 /Tests/Marketplace/approved_usecases.json @demisto/content-leaders @michalgold
-/Packs/CommonScripts/Scripts/SetGridField/* @yaakovi
+
+# Important integrations
+/Packs/Jira/Integrations/JiraV2/* @demisto/content-leaders
+/Packs/ServiceNow/Integrations/ServiceNowv2/* @demisto/content-leaders
+/Packs/EWS/Integrations/EWSv2/* @demisto/content-leaders
+/Packs/Slack/Integrations/Slack/* @demisto/content-leaders
+/Packs/SplunkPy/Integrations/SplunkPy/* @demisto/content-leaders
+/Packs/Active_Directory_Query/Integrations/Active_Directory_Query/* @demisto/content-leaders
```
Comment (anara123) [inline .github/CODEOWNERS:11]: add 
VirusTotal
ArcSight ESM v2

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,3 +1,14 @@
+# Security related
```
Comment (yaakovi) [inline .github/CODEOWNERS:1]:
```suggestion
# Marketplace related
```

File: .github/CODEOWNERS (hunk 2 above)
Comment (yaakovi) [reply .github/CODEOWNERS:11]: maybe QRadar? we did not have a bad experience with it but it is widely used.

**Comment by bakatzir** [comment]
Added your reqs, also added the `SearchIncidentsV2` script.
we can give it a week or two, and if too many notifications we can divide it between us per speciallity, at least 2 owners per content entity.

**Review by glicht — APPROVED** [review]
Looks good. I am not sure about the list but I guess we can improve as we gain experience.

**Pull request merged**